cdxev/__main__.py

```diff
@@ -297,6 +298,16 @@ def create_validation_parser(
         ),
         type=str,
     )
+    parser.add_argument(
+        "--plausability-check",
+        metavar="<plausability-check>",
+        choices=["yes", "y"],
+        help=(
+            "y/yes if the plausibility of the bom-refs in the"
+            "sbom should also be checked"
+        ),
```

Comment:
Why this additional choice? Why not providing the flag `plausibility-check` means `true`?